src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -800,14 +800,14 @@ def operation_if_column_snql(
             Function(
                 "and",
                 [
-                    aggregate_filter,
                     Function(
                         "equals",
                         [
                             Column(resolve_tag_key(use_case_id, org_id, if_column)),
                             resolve_tag_value(use_case_id, org_id, if_value),
                         ],
                     ),
+                    aggregate_filter,
```

Comment:
I'm going to be using `operation_if_column_snql` in a test and this re-ordering is necessary for comparison even though it is an `and` operation.